tests/bud.bats

```diff
@@ -4547,17 +4547,17 @@ EOM
 }
 
 @test "bud arg and env var with same name" {
-  _prefetch centos:8
+  _prefetch busybox
```

Comment:
same for the other image please make sure to use quay.io/libpod images if we already change them.